examples/slow_mouse.lua

```diff
@@ -0,0 +1,102 @@
+--
+-- SPDX-FileCopyrightText: (c) 2025 Jieming Zhou <qrsikno@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+-- example of a slow mouse driver
+local hid = require("luahid") 
+local function dummy_submitx(value)
+	-- pass
+end
+local function dummy_submity(value)
+	-- pass
+end
+local slow_mouse_driver = {
```

Comment:
please, add readme entry for describing this API and every field in this table..